data/brands/amenity/cafe.json

```diff
@@ -3328,6 +3342,20 @@
         "takeaway": "yes"
       }
     },
+    {
+      "displayName": "The Bean",
+      "locationSet": {
+        "include": ["new_york_city.geojson"]
+      },
+      "tags": {
+        "amenity": "cafe",
+        "brand": "The Bean",
+        "brand:wikidata": "Q122209700",
```

Comment:
It looks like this brand only has 3 locations in NYC: https://www.thebean.nyc/hours-locations

In the future, I'd try to stick with adding brands with at least 10 locations.